### PR TITLE
New package: n-t-roff-sc, a maintained fork of sc 7.16

### PR DESCRIPTION
**Pull request by microsounds**
I'm proposing ~~switching sc's upstream~~ from the original 20 year old 7.16 source code on ibiblio.org to a maintained fork [n-t-roff/sc](https://github.com/n-t-roff/sc) on github.

I changed my proposal to adding this maintained fork as a separate package `n-t-roff-sc` alongside `sc`

## Why?
~~This commit~~ bumps version number to 7.16_1.1.2 from n-t-roff's fork of sc that fixes sc compile-time errors ~~that were previously being patched by this repo~~ and has none of the run-time issues I've been experiencing with the currently packaged sc 7.16 that make it unusable for me, specifically:
* When opening sc, it hangs and does not redraw the TUI unless I resize the Termux window by changing device orientation or opening and closing the on screen keyboard.
* sc does not allow me to populate any cells without giving a syntax error as described by issue #5889
* When opening *.sc files, every single line within the .sc file throws a syntax error and no .sc files can be read.

This maintained fork fixes all these issues for me, it compiles and works perfectly for me under `arm` Termux 0.118.0 with the following changes in the included patch.
* `Makefile.in` uses hard links during the build process, causing the build to fail under Termux, changing these for symlinks lets the build finish successfully.

### CODE REVIEW

File: packages/sc/build.sh [hunk 1]
```diff
@@ -1,14 +1,12 @@
-TERMUX_PKG_HOMEPAGE="http://www.ibiblio.org/pub/Linux/apps/financial/spreadsheet/!INDEX.html"
+TERMUX_PKG_HOMEPAGE="https://github.com/n-t-roff/sc"
 TERMUX_PKG_DESCRIPTION="A vi-like spreadsheet calculator"
 TERMUX_PKG_LICENSE="Public Domain"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=7.16
-TERMUX_PKG_REVISION=4
-TERMUX_PKG_SRCURL=http://www.ibiblio.org/pub/Linux/apps/financial/spreadsheet/sc-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=1997a00b6d82d189b65f6fd2a856a34992abc99e50d9ec463bbf1afb750d1765
+TERMUX_PKG_VERSION=7.16_1.1.2
```
Comment (xtkoba) [inline packages/sc/build.sh:5]: Please use a more saner version string, in which an underscore (`_`) should be avoided. I propose something like:

```bash
_VERSION=7.16_1.1.2
TERMUX_PKG_VERSION=${_VERSION//_/-}
```

**Review by xtkoba — COMMENTED** [review]
Also I propose that this fork be given another package name (than `sc`).

**Comment by microsounds** [comment]
Changing the package name is fine, I'm not sure what kind of name would be appropriate, the maintained fork still looks and acts like sc 7.16 unlike sc-im which has had major changes including keybinds.

Also I realize I left the corrected version number in the URL that fetches upstream and it 404'd, fixed that now.

**Comment by xtkoba** [comment]
`n-t-roff-sc` would be appropriate, which is adopted <a href="https://www.opensourceagenda.com/projects/n-t-roff-sc/">here</a>. Then I suggest putting the following lines in `build.sh` of `n-t-roff-sc` package:

```bash
TERMUX_PKG_CONFLICTS="sc"
TERMUX_PKG_REPLACES="sc"
TERMUX_PKG_PROVIDES="sc"
```

**Comment by microsounds** [comment]
Sounds good, I made `n-t-roff-sc` a distinct package from vanilla `sc`.

It seems the build process expects a more featured `./configure` script than the one in this package, I went ahead and let it ignore unsupported flags.

**Pull request merged**